api/controllers/reports.js

```diff
@@ -0,0 +1,232 @@
+'use strict';
+
+var _ = require('lodash');
+var async = require('async');
+var User = require('../../models/User');
+var Hours = require('../../models/Hours');
+var weeksSinceEpoch = require('../../lib/util').weeksSinceEpoch;
+
+/**
+ * Gets hours for an epoch window (optionally), and returns them in flat layout.
+ */
+
+var getHours = function (epochSubtractor, callback) {
```

Comment:
I would consider making this a `static` on the Hours model, since this is basically a new query function for the Hours model.